model_weight: "/home/roboticslab/Storage/YoloTag/runs/detect/train6/weights/best.pt"
b_coeff: [2.72133124e-05, 5.44266249e-05, 2.72133124e-05]
a_coeff: [1.98519079, -0.98529965]
order: 2
conf_thrs: 0.5

Tags:
 "1": [0.80, 6.0, 0.724]
 "2": [1.44, 6.0, 0.724]
 "3": [2.045, 6.0, 0.724]
 "4": [2.579, 6.0, 0.724]
 "5": [3.175, 6.0, 0.724]
 "6": [3.747, 6.0, 0.724]
 "7": [4.343, 6.0, 0.724]
 "8": [4.877, 6.0, 0.724]


camera_matrix: [537.292878, 0.000000, 427.331854, 0.000000, 527.000348, 240.226888, 0.000000, 0.000000, 1.000000]
dist_coeffs: [0.004974, -0.000130, -0.001212, 0.002192, 0.000000]